index.html

```diff
@@ -1363,14 +1377,21 @@ <h4>Break calibration</h4>
           at runtime when this mitigation is running continuously. Any attempts to recalibrate
           will similarly be mitigated against.
         </p>
-        <div class="note">
+        <aside class="note">
+          This mitigation, if applied too often, can deteriorate the pressure state detection reliability.
+          Therefore it is targeted for longer calibration attack processes.
+          Implementers are advised to apply the mitigation to a randomized time value in milliseconds
+          within a range between 120000 and 240000 (2 and 4 minutes).
+          Faster calibration processes can be mitigated by [=rate obfuscation=] mitigation.
```

Comment:
I don't think it is clear what "faster calibration processes" refers to.

When can you calibrate fast? and when can you calibrate slowly?

You calibrate when you have focus and we need to make sure to somewhat invalidate that calibration. We dont even have to do it between a special time window, we could even do it whenever another page gains indirect focus and starts receiving telemetry data. - that should work just fine now that I am thinking about it - or just use slighly different thresholds per origin

Reply:
Faster calibration processes, I m thinking of something like you are doing with the demo...
We chnage the state very fast (5-10s) and we are trying to figure out in a short time period what state belongs to what load.
slower I was thinking that you are one in a while trying to increase the work load... every minute... 
So the frequency of load change would be really low.

Reply:
but I think it is very confusing wording because they reader doesn't know about the demo etc